bspump/mysql/binlogsource.py

```diff
@@ -99,10 +100,13 @@ def __init__(self, app, pipeline, connection, id=None, config=None):
 		self.ServerId = int(self.Config['server_id'])
 		self.LogFile = self.Config['log_file']
 		self.LogPos = int(self.Config['log_pos'])
+		self.AutoPosition = bool(self.Config['auto_position'])
```

Comment:
I think `self.Config.getboolean('auto_position')` is more preferable choice here.

Reply:
Thanks, done.